apps/src/codebridge/InfoPanel/HelpAndTips.tsx

```diff
@@ -10,8 +10,13 @@ const HelpAndTips: React.FunctionComponent = () => {
   const referenceLinks = useAppSelector(
     state => state.lab.levelProperties?.referenceLinks
   );
+  const helpVideos = useAppSelector(
+    state => state.lab.levelProperties?.helpVideos
+  );
+  const helpVideo = helpVideos ? helpVideos[0] : null;
```

Comment:
Do we need to be able to support showing more than one video? The `HelpTabContents` component is only expecting one, the levelbuilder field seems to only allow you to select one, but the existing level property for videos is an array...